documentation/docs/steps/transportRequestUploadFile.md

```diff
@@ -64,6 +64,82 @@ The properties can also be configured on a per-step basis:
 
 The parameters can also be provided when the step is invoked. For examples see below.
 
+## CTS Uploads
+
+In order to be able to upload the application, it is required to build the application, e.g. via `npmExecute`
+or `mtaBuild`. The content of the app needs to be provided in a folder named `dist` in the root level of the project.
+Although the name of the step `transportRequestUploadFile` might suggest something else, in this case a folder needs
+to be provided. The application, which is provided in the `dist` folder is zipped and uploaded by the fiori toolset
+used for performing the upload.
+
+For `CTS` related uploads we use a node based toolset. When running in a docker environment a standard node
+image can be used. In this case the required deploytool dependencies will be installed prior to the deploy.
+It is also possible to provide a docker image which already contains the required deploy tool
+dependencies (`config.changeManagement.cts.nodeDocker.image`). In this case an empty list needs to be provided
+as `config.changeManagement.cts.deployToolDependencies`. Using an already pre-configured docker image speeds-up
```

Comment:
Can't we infer this from the dockerImage being provided? Why does the user have to configure an empty list here?

Reply:
With this approach here it is also possible to provide an image e.g. derived from the standard node image with some added company specific certificates only but without the fiori tools. That seems to be a valid use case for me since a customer can create one common customer specific base image with only the certificates added which can be used for several purposes. This approach here provides more (IMO reasonable) degrees of freedom to the user.

Reply:
... yes.